app/views/publish/users/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<% page_title = t("page_titles.users.edit") %>
+<%= content_for :page_title, title_with_error_prefix(page_title, @user_form.errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_user_path) %>
+<% end %>
+
+<%= form_with(model: @user_form, url: publish_provider_user_path, method: :put, local: true) do |f| %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <%= f.govuk_error_summary %>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-fieldset__heading">
+            <span class="govuk-caption-l"><%= @provider_user.full_name %></span>
+            <%= t("page_titles.users.edit") %>
+          </h1>
+        </legend>
+        <%= f.govuk_text_field :first_name, label: { text: "First name", size: "s" }, width: "two-thirds" %>
+        <%= f.govuk_text_field :last_name, label: { text: "Last name", size: "s" }, width: "two-thirds" %>
+        <%= f.govuk_text_field :email, label: { text: "Email address", size: "s" }, width: "full" %>
+        <%= f.govuk_submit %>
+      </fieldset>
+<% end %>
```

Comment:
Looks out of place?

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/3185/commits/4667ad38b81d1c1aefdae8b5ad43584bb47e0768